unity/Assets/Scripts/PhysicsRemoteFPSAgentController.cs

```diff
@@ -6936,129 +6936,149 @@ public bool objectIsCurrentlyVisible(SimObjPhysics sop, float maxDistance)
             }
 
             return allVisible;
-
         }
 
-        public void PositionsFromWhichItemIsInteractable(ServerAction action) {
-
-            //default to increments of 30 for horizon
-            if(action.horizon == 0)
-            {
-                action.horizon = 30;
-            }
-
-            //check if horizon is a multiple of 5
-            if(action.horizon % 5 != 0)
-            {
-                errorMessage = "Horizon value for PositionsFromWhichItemIsInteractable must be a multiple of 5";
-                actionFinished(false);
-                return;
-            }
-
-            if(action.horizon < 0 || action.horizon > 30)
-            {
-                errorMessage = "Horizon value for PositionsFromWhichItemIsInteractable must be in range [0, 30] inclusive";
+        // Get the poses with which the agent can interact with 'objectId'
+        // @rotations: if rotation is not specified, we use rotateStepDegrees, which results in [0, 90, 180, 270] by default.
+        public void GetInteractablePositions(
+            string objectId,
+            Vector3[] positions = null,
+            float[] rotations = null,
+            float[] horizons = null,
+            bool[] standings = null
+        ) {
```

Comment:
These allow you to ONLY check for certain positions, rotations, horizons, or standing booleans (i.e., standing is `True` or `False`).  These should be super easy to use in Python with linspaces and ranges (which are inclusive at both ends by default, unlike ranges):
```python
import numpy as np
# ... controller setup  ...
controller.step('GetInteractablePositions'
    objectId=<MY_OBJECT_ID>,
    rotations=range(0, 360, 10),
    horizons=np.linspace(-30, 60, 20),
    standings=[True]
)
```

The soon to be deprecated `PositionsFromWhichItemIsInteractable` used to take `positions` as a position filter but didn't allow for any others. (It also took `horizon` as a parameter, which worked like a horizon incremented).